app/assets/scripts/views/field-report-form.js

```diff
@@ -0,0 +1,1235 @@
+  }
+
+  renderStepper () {
+    const step = this.state.step;
+    const items = [
+      'Basic Information',
+      'Numeric Details',
+      'Actions Taken',
+      'Planned Response',
+      'Contacts'
+    ];
+    return (
+      <ol className='stepper'>
+        {items.map((o, idx) => {
+          idx += 1;
+          const classes = c('stepper__item', {
+            'stepper__item--complete': step > idx,
+            'stepper__item--current': step === idx
+          });
+          return <li key={o} className={classes}><a href='#' onClick={this.onStepperClick.bind(this, idx)}><span>{o}</span></a></li>;
+        })}
+      </ol>
+    );
+  }
+
+  renderStep1 () {
+    return (
+      <Fold title='Basic Information'>
+        <FormInput
+          label='Summary *'
+          type='text'
+          name='summary'
+          id='summary'
+          description={<div className='form__description'><p>Write a short summary of the report's topic. It will be used as the subject of the e-mail notification,
+          later as the tittle of the RSS feed and possibly as the text message on mobile phones.</p><em>Example: 250 dead after an earthquake in Indonesia</em></div>}
+          value={this.state.data.summary}
+          onChange={this.onFieldChange.bind(this, 'summary')}
+          autoFocus >
+
+          <FormError
+            errors={this.state.errors}
+            property='summary'
+          />
+
+        </FormInput>
+
+        <div className='form__group'>
+          <label className='form__label'>Countries *</label>
+          <p className='form__description'>Seach for the affected country. You can select more than one.</p>
+          <Select
+            name='countries'
+            value={this.state.data.countries}
+            onChange={this.onFieldChange.bind(this, 'countries')}
+            options={formData.countries}
+            multi />
+
+          <FormError
+            errors={this.state.errors}
+            property='countries'
+          />
+        </div>
+
+        <FormRadioGroup
+          label='Status *'
+          name='status'
+          options={formData.status}
+          selectedOption={this.state.data.status}
+          onChange={this.onFieldChange.bind(this, 'status')}>
+          <FormError
+            errors={this.state.errors}
+            property='status'
+          />
+        </FormRadioGroup>
+
+        <div className='form__hascol form__hascol--2'>
+          <FormSelect
+            label='Disaster Type *'
+            name='disaster-type'
+            id='disaster-type'
+            options={formData.disasterType}
+            value={this.state.data.disasterType}
+            onChange={this.onFieldChange.bind(this, 'disasterType')} >
+            <FormError
+              errors={this.state.errors}
+              property='disasterType'
+            />
+          </FormSelect>
+
+          <FormSelect
+            label='Event'
+            name='event'
+            id='event'
```

Comment:
Eventually (very soon hopefully) this can turn into an autocomplete form that will query the API. Obviously we need to keep it like this for now, but just to keep that in mind.